### PR TITLE
File import

### LINKED ISSUE #10: Importing backups in firefox doesn't work
When the file select dialog comes up, it closes the popup and stops processing js.

### PR DESCRIPTION
**Pull request by mymindstorm**
closes #10 

I haven't tested this in Chrome

### CODE REVIEW

File: import.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<html>
+<head>
+	<meta charset="utf-8"/>
```
Comment (Sneezry) [inline import.html]: all tabs should be changed to spaces

File: import.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	<script src="build/models/interface.js"></script>
+	<script src="build/models/key-utilities.js"></script>
+	<script src="build/models/otp.js"></script>
+	<script src="build/models/storage.js"></script>
+	<script src="build/models/dropbox.js"></script>
+</head>
+<body>
+<div id="authenticator">
+	<input id="import_file" type="file" v-on:change="importFile($event, true)" accept="application/json"></input>
+
```
Comment (Sneezry) [inline import.html]: extra empty line

File: import.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+</div>
+</body>
+<script src="build/ui/add-account.js"></script>
```
Comment (Sneezry) [inline import.html]: script tabs should be in body

**Comment by mymindstorm** [comment]
Changes made

File: popup.html [hunk 4]
```diff
@@ -120,8 +120,11 @@
             <div v-show="info === 'export'">
                 <div id="security_warning">{{ i18n.export_info }}</div>
                 <a download="authenticator.json" v-bind:href="exportFile" id="download_backup">{{ i18n.download_backup }}</a>
-                <label id="upload_backup" for="import_file">{{ i18n.import_backup }}</label>
-                <input id="import_file" type="file" v-on:change="importFile($event)" accept="application/json">
+                <div v-if="navigator.userAgent.indexOf('Chrome') !== -1">
```
Comment (Sneezry) [inline popup.html]: it doesn't work on my side, Chrome opens import.html as well. No idea why...

Comment (Sneezry) [reply popup.html]: `navigator` is not set in Vue. It is a scope issue. In origin JS, `navigator` is in `window` namespace (`window.navigator`), but in Vue template, the scope is Vue instance, so `navigator` points to `authenticator.navigator` which is undefined. We have to wrap a function in ui to check UA.

Comment (mymindstorm) [reply popup.html]: No wonder v-if was giving me such a hard time.

**Comment by mymindstorm** [comment]
That should fix it.

File: popup.html [hunk 5]
```diff
@@ -120,8 +120,11 @@
             <div v-show="info === 'export'">
                 <div id="security_warning">{{ i18n.export_info }}</div>
                 <a download="authenticator.json" v-bind:href="exportFile" id="download_backup">{{ i18n.download_backup }}</a>
-                <label id="upload_backup" for="import_file">{{ i18n.import_backup }}</label>
-                <input id="import_file" type="file" v-on:change="importFile($event)" accept="application/json">
+                <div v-if="isChrome">
```
Comment (Sneezry) [inline popup.html]: `isChrome()`, or it will always be true

File: popup.html [hunk 6]
```diff
@@ -120,8 +120,11 @@
             <div v-show="info === 'export'">
                 <div id="security_warning">{{ i18n.export_info }}</div>
                 <a download="authenticator.json" v-bind:href="exportFile" id="download_backup">{{ i18n.download_backup }}</a>
-                <label id="upload_backup" for="import_file">{{ i18n.import_backup }}</label>
-                <input id="import_file" type="file" v-on:change="importFile($event)" accept="application/json">
+                <div v-if="isChrome()">
+                        <label id="upload_backup" for="import_file">{{ i18n.import_backup }}</label>
+                        <input id="import_file" type="file" v-on:change="importFile($event)" accept="application/json"/>
+		</div>
```
Comment (Sneezry) [inline popup.html]: Change tabs to spaces in this line and the next line

**Review by Sneezry — APPROVED** [review]

**Pull request merged**